changelogutils/changelog_test.go

```diff
@@ -44,6 +44,8 @@ var _ = Describe("ChangelogTest", func() {
 			Expect(getProposedTag("v1.0.0-beta1", tmpDir, "v1.0.0-beta2")).To(BeNil())
 			Expect(createSubdirs(changelogDir, "v1.0.0-rc1")).To(BeNil())
 			Expect(getProposedTag("v1.0.0-beta2", tmpDir, "v1.0.0-rc1")).To(BeNil())
+			Expect(createSubdirs(changelogDir, "v1.0.0-rc2")).To(BeNil())
+			Expect(getProposedTag("v1.0.0-rc1", tmpDir, "")).NotTo(BeNil())
```

Comment:
Test to show that the deprecated `changelogutils.GetProposedTag()` method (not using changelogValidator), has the issue where it throws an error if `beta1, rc1` and `rc2` changelogs exist in the same directory.